lib/pbench/test/unit/server/test_schema.py

```diff
@@ -128,15 +130,16 @@ def ok(username: str) -> User:
     @pytest.mark.parametrize(
         "ptype,value",
         (
-            (ParamType.STRING, {"not": "string"}),  # dict is not string
-            (ParamType.JSON, (1, False)),  # tuple is not JSON
+            (ParamType.ACCESS, ["foobar"]),  # ACCESS is "public" or "private"
+            (ParamType.ACCESS, 0),  # ACCESS must be a string
             (ParamType.DATE, "2021-06-45"),  # few months have 45 days
             (ParamType.DATE, "notadate"),  # not valid date string
             (ParamType.DATE, 1),  # not a string representing a date
+            (ParamType.INT, "a"),  # can not convert to an int
+            (ParamType.JSON, (1, False)),  # tuple is not JSON
+            (ParamType.STRING, {"not": "string"}),  # dict is not string
             (ParamType.USER, False),  # not a user string
             (ParamType.USER, "xyzzy"),  # not a defined username
```

Comment:
There's no case for either `ParamType.KEYWORD` or `ParamType.LIST`...should there be?

Reply:
[No.](https://github.com/distributed-system-analysis/pbench/pull/2855#discussion_r882046133)